config/ppo/Sorter_curriculum.yaml

```diff
@@ -0,0 +1,105 @@
+behaviors:
+  Sorter:
+    trainer_type: ppo
+    hyperparameters:
+      batch_size: 512
+      buffer_size: 40960
+      learning_rate: 0.0003
+      beta: 0.005
+      epsilon: 0.2
+      lambd: 0.95
+      num_epoch: 3
+      learning_rate_schedule: linear
+    network_settings:
+      normalize: False
+      hidden_units: 128
+      num_layers: 2
+      vis_encode_type: simple
+    reward_signals:
+      extrinsic:
+        gamma: 0.99
+        strength: 1.0
+    keep_checkpoints: 5
+    max_steps: 100000000
```

Comment:
We can reduce this. I think we should try to optimize training time a bit before we waste training hours on CI